### ISSUE TITLE
Some python related issues

### ISSUE THREAD

**Issue by mbkumar** [zhanghe9704/tpsa#14]
I found a few issues on the python side.

1) Examples:
    a) Unnecessary imports: 01 and 06 don't need to have `import pybind11` 
        statement. Please delete those import statements. 
        In the instructions on README.rd, remove the below install instructions. They 
        are automatically downloaded when someone installs tpsa with pip.
        ```bash
        pip install pybind11 
        pip install scikit-build-core
        ```
   b) Segfaults: 02, 03, 04 are failing with segfaults on Ubuntu 22.04. 

2) Carriage return chars in `tpsa_python.cc` file.

**Comment by zhanghe9704** [comment]
1. Examples:
 a） Removed the import statements in the examples and the installation commands in README
 b)   I found the Segfaults happen when running the examples by "python 03_substitute.py". If I run it inside the python environment using "exec(open("03_substitute.py").read())", the segfaults won't happen until I call "exit()" to quit from python. 
So probably the segfaults are caused by some destructors when Python is quitting and trying to release memory. Unfortunately, I still don't know what's happening exactly in this process. But I found the segfaults can be avoided by simply enlarging the memory pool size. For now, the examples are fixed by initializing a larger memory pool when calling da_init().

2. Line breaks are changed into Unix style.
